### PR TITLE
More fixes for styleguide

### PR DESCRIPTION
**Pull request by hbillings**
Closes the remainder of #1115, plus a few other bugs I found while looking at those.

### CODE REVIEW

**Comment by jseppi** [comment]
@hbillings: opened https://github.com/18F/calc/pull/1210 against this to remove the tablesort code since it is unused and won't be used

File: styleguide/templates/styleguide.html [hunk 1]
```diff
@@ -238,57 +236,20 @@ <h3 class="example-heading">Example</h3>
...
-    background of the error alert because the red background would make it
-    difficult to examine the amount of information being displayed.
+    but add a stroke and the CALC standard border radius.
   </p>
 
   {% example %}
   <div class="alert alert-error" role="alert">
-    <h3>2 rows have errors</h3>
-    <p>The rows below appear to be invalid and <strong>will be discarded</strong> when you upload your price list. If you'd like, you may correct these rows in your original spreadsheet and <a href="foo">try uploading it again</a>.</p>
-    <table>
```
Comment (jseppi) [inline styleguide/templates/styleguide.html]: Will we have a new example for the table of error rows in a price list upload?

File: styleguide/templates/styleguide.html [hunk 2]
```diff
@@ -307,110 +268,57 @@ <h3 class="example-heading">Example</h3>
...
-          <td>$65.72</td>
+          <td class="number">7</td>
+          <td class="number currency">65.72</td>
         </tr>
       </tbody>
     </table>
   {% endexample %}
-  <p>Sortable tables are used for displaying large quantities of data and are usually sorted by default on the price column. <strong>Make sure to include the correct classes on columns containing numbers, or they will be sorted as strings.</strong> We use <a href="https://www.npmjs.com/package/jquery-tablesort">jQuery Tablesort</a> for this. (The exception is on the CALC homepage, where sorting is built into the table with D3.) Sortable tables look like this:</p>
-  {% example %}
-    <table class="table-sortable">
```
Comment (jseppi) [inline styleguide/templates/styleguide.html]: :pour-one-out:

**Comment by jseppi** [comment]
Sweet, thanks! I don't think we are using or have plans to use the success-styled input, so should it be removed from the guide?

**Comment by hbillings** [comment]
Removed! I left the CSS in there in case we want to use it for validation or soemthing in the future.

**Comment by jseppi** [comment]
Maybe forgot to push the removal?

**Comment by hbillings** [comment]
derp

**Pull request merged**